tools/astarte_import/lib/astarte/import.ex

```diff
@@ -125,13 +126,16 @@ defmodule Astarte.Import do
   end
 
   defp xml_event({:startElement, _uri, _l_name, {_prefix, 'credentials'}, attr}, _loc, state) do
-    with inhibit_request_string = get_attribute(attr, 'inhibit_request', "false"),
+    with {:ok, inhibit_request_string} = get_attribute(attr, 'inhibit_request', "false"),
          {:ok, false} <- to_boolean(inhibit_request_string),
-         {:ok, cert_serial} <- fetch_attribute(attr, 'cert_serial'),
-         {:ok, cert_aki} <- fetch_attribute(attr, 'cert_aki'),
-         {:ok, first_credentials_string} <- fetch_attribute(attr, 'first_credentials_request'),
-         {:ok, first_credentials_request, 0} <- DateTime.from_iso8601(first_credentials_string),
-         {:ok, last_creds_ip_string} <- fetch_attribute(attr, 'last_credentials_request_ip'),
+         {:ok, cert_serial} <- get_attribute(attr, 'cert_serial', ""),
+         {:ok, cert_aki} <- get_attribute(attr, 'cert_aki', ""),
+         {:ok, first_credentials_string} <-
+           get_attribute(attr, 'first_credentials_request', "#{DateTime.utc_now()}"),
```

Comment:
It is possible for devices to not have `first_credentials_request` set: as an example, a device could be registered in Astarte but still not have requested any credentials. So, maybe it would be better to default this to `nil` in order not to import data that is not there